static_frame/test/unit/test_index.py

```diff
@@ -591,8 +591,8 @@ def test_index_go_f(self) -> None:
         idx1 = IndexAutoFactory.from_optional_constructor(3,
                 default_constructor=IndexGO)
         idx1.append(3) # type: ignore
-        post = idx1._loc_to_iloc(np.array([True, False, True, False]), 2) #type: ignore
-        self.assertEqual(post.tolist(), [2, 4]) #type: ignore
+        post = idx1._loc_to_iloc(np.array([True, False, True, False]))
+        self.assertEqual(post.tolist(), [True, False, True, False]) #type: ignore
```

Comment:
Yeah `_loc_to_iloc` can return boolean arrays. It's actually more efficient too since it just passes through. Honestly I'm not sure what this old behavior was doing, as it relates to `IndexLevel._loc_to_iloc` and `offset`s